src/platform/haswell/include/platform/lib/memory.h

```diff
@@ -133,7 +135,11 @@ void platform_init_memmap(void);
 #define SOF_STACK_BASE		(DRAM0_BASE + DRAM0_SIZE)
 #define SOF_STACK_END		(SOF_STACK_BASE - SOF_STACK_TOTAL_SIZE)
 
-#define MAILBOX_BASE			(SOF_STACK_END - MAILBOX_SIZE)
+/* Virtual threadptr */
+#define SOF_VIRTUAL_THREAD_SIZE	0x20
```

Comment:
4 bytes would be enough, but I assume it's some kind of precaution?

Reply:
Alignment and support for more cores.

Reply:
But as for the alignment I believe its already done by:

```
 . = ALIGN (32);
    _virtual_thread_start = ABSOLUTE(.);
    . = . + SOF_VIRTUAL_THREAD_SIZE;
    _virtual_thread_end = ABSOLUTE(.);
```

For multi core I believe you want later one do sth like thread_ptr[core_id] right? I said "later" since as for now I don't see support for that.

So to me the size can be 0x10.